docs/validators/manage/vrf-key.mdx

```diff
@@ -0,0 +1,44 @@
+---
+description: Add a VRF key to your profile. 
+title: Add or change VRF key
+---
+import profile from './assets/profile.png';
+import addStepOne from './assets/add-vrf-key-1.png';
+import addStepTwo from './assets/add-vrf-key-2.png';
+
+## Overview
+
+This guide demonstrates how to add a *VRF key* to your profile. The VRF key is used to [generate random beacon](https://github.com/ronin-chain/REPs/blob/main/REP-0010/REP-0010.md#compute-random-beacon). If you're an active Ronin Governing Validator, you need to have this key added.
+
+## Generate a VRF key pair
+
+You can generate VRF Key pair using the Ronin Random Beacon Source Code by running the following commands:
```

Comment:
## Prerequisites 

Ensure you have installed `Go`, the Go version should be 1.19 or higher.